settings.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
 rootProject.name = 'graphql-tools'
 include(':config:checkstyle')
 include(':graphql-cli')
+include(':graphql-bal-service')
```

Comment:
This name does not express the purpose of the sub module. `graphql-schema-file-generator` might be a good fit.

Reply:
changed in https://github.com/ballerina-platform/graphql-tools/pull/118/commits/53b676654558e4f4795138d17b0bde15524498c3